tsconfig.eslint.json

```diff
@@ -1,5 +1,5 @@
 {
   "extends": "./tsconfig.json",
-  "include": ["/**/*.ts"],
+  "include": ["**/*.ts"],
```

Comment:
Linting was not working on windows, so all the `./` removal is to appease windows.